agent/rpm/rpm.mk

```diff
@@ -23,20 +23,20 @@ RPMSPEC = ${HOME}/rpmbuild/SPECS
 ###########################################################################
 
 # building on COPR.
-# version and sha1 have to be provided by the including Makefile.
+# VERSION and sha1 have to be provided by the including Makefile.
 # The including Makefile also has to provide the srpm target.
 
 .SECONDEXPANSION:
-$(RPMSRPM)/$(prog)-$(version)-$(seqno)$(sha1).src.rpm: srpm
+$(RPMSRPM)/$(prog)-$(VERSION)-$(seqno)g$(sha1).src.rpm: srpm
 
 ifdef COPR_USER
 _copr_user = ${COPR_USER}
 else
 _copr_user = ${USER}
 endif
 
-copr\
-copr-test:	$(RPMSRPM)/$(prog)-$(version)-$(seqno)$(sha1).src.rpm
-	copr-cli build ${CHROOTS} $(_copr_user)/$(subst copr,pbench,$@) $(RPMSRPM)/$(prog)-$(VERSION)-$(seqno)g$(sha1).src.rpm
+copr \
+copr-test:	$(RPMSRPM)/$(prog)-$(VERSION)-$(seqno)g$(sha1).src.rpm
```

Comment:
Maybe change the parens to braces to make things uniform? l.30, 39 and 40.

Reply:
I thought you wanted to go `()-->{}` rather than the other way around? It doesn't matter much though.

Reply:
This is interesting, because the new commit changes braces at line 40 to parens and parens at line 23 in `rpm.mk` to braces. I guess this makes each command internally consistent in, but is that better, worse, or irrelevant? 🍤